### PR TITLE
Update client with basic Roman support

### PR DESCRIPTION
**Pull request by eslavich**
This PR adds Roman support to the CRDS client.  It includes relevant files from @jaytmiller and @jbdoggett's [roman-spike branch](https://github.com/spacetelescope/crds/pull/762), with the following updates:

- Update name of romancal package and RomanDataModel base class
- Import ValidationWarning from stdatamodels
- Add NotImplementedError to some of the stubbed out methods (filekind_to_keyword, get_exptypes)
- Minor docstring editing
- Remove commented-out FITS support
- Regenerate crds/roman/specs/combined_specs.json
- Update .tpns with detector and filter values from the current romancal core schema
- Add env variable scripts for dev, test, and int deployments
- Use Roman-ish filenames in the docstring tests

### CODE REVIEW

File: crds/roman/specs/combined_specs.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "all":{
+        "all":{
+            "extra_keys":null,
+            "file_ext":".fits",
```
Comment (jbdoggett) [inline crds/roman/specs/combined_specs.json:5]: .fits --> .asdf

File: crds/roman/specs/combined_specs.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            "suffix":"xxx",
+            "text_descr":"All instruments and reference file types.",
+            "tpn":"all_all.tpn",
+            "unique_rowkeys":null
+        }
+    },
+    "wfi":{
+        "all":{
+            "extra_keys":null,
+            "file_ext":".fits",
```
Comment (jbdoggett) [inline crds/roman/specs/combined_specs.json:18]: .fits -->.asdf

**Review by jbdoggett — CHANGES_REQUESTED** [review]

File: crds/roman/specs/combined_specs.json (hunk 1 above)
Comment (eslavich) [reply crds/roman/specs/combined_specs.json:5]: Oops, thanks!

Comment (eslavich) [reply crds/roman/specs/combined_specs.json]: Addressed in latest commit

File: crds/roman/specs/combined_specs.json (hunk 2 above)
Comment (eslavich) [reply crds/roman/specs/combined_specs.json]: Addressed in latest commit

**Pull request merged**